packages/angular/ssr/test/npm_package/BUILD.bazel

```diff
@@ -0,0 +1,35 @@
+load("@build_bazel_rules_nodejs//:index.bzl", "nodejs_binary")
+load("@npm//@bazel/jasmine:index.bzl", "jasmine_node_test")
+load("//tools:defaults.bzl", "ts_library")
+
+ts_library(
+    name = "unit_test_lib",
+    testonly = True,
+    srcs = glob(["**/*.ts"]),
+    deps = [
+        "@npm//@bazel/runfiles",
+        "@npm//@types/diff",
+    ],
+)
+
+jasmine_node_test(
+    name = "test",
+    srcs = [":unit_test_lib"],
+    data = [
+        "THIRD_PARTY_LICENSES.txt.golden",
+        "//packages/angular/ssr:npm_package",
+        "@npm//diff",
+    ],
+)
+
+nodejs_binary(
```

Comment:
Suggestion: Would Skylib's `diff_test` be a more out-of-the-box solution here?

https://github.com/bazelbuild/bazel-skylib/blob/main/docs/diff_test_doc.md

Reply:
I did try to use it, but it always fails with a strange diff error that I am not understanding

```
==================== Test output for //packages/angular/ssr/test/npm_package:critters_license_test:
174c174
< 
---
>  
177c177
< 
---
>  
FAIL: files "packages/angular/ssr/test/npm_package/THIRD_PARTY_LICENSES.txt.golden" and "packages/angular/ssr/third_party/critters/THIRD_PARTY_LICENSES.txt" differ.
```

Reply:
Ahh never mind it was spaces.